static/chat.delta.desktop.appdata.xml

```diff
@@ -25,7 +25,7 @@
   <launchable type="desktop-id">chat.delta.desktop.desktop</launchable>
   <screenshots>
     <screenshot type="default">
-      <image>https://github.com/deltachat/deltachat-desktop/raw/master/screenshot.png</image>
+      <image>https://github.com/deltachat/deltachat-desktop/raw/master/README_ASSETS/screenshot.png</image>
```

Comment:
how about changing this to an absolute commit to prevent a missing image when its path should change again?

Reply:
But then we need to update the path on every new screenshot push